models/zeroex/optimism/zeroex_optimism_schema.yml

```diff
@@ -144,4 +144,50 @@ models:
       - *contract_address
       - &fills_within
         name: fills_within
-        description: "fills in then multihop, if present"          
\ No newline at end of file
+        description: "fills in then multihop, if present"
+
+  - name: zeroex_optimism_native_fills 
+    meta:
+      blockchain: optimism
+      project: zeroex
+      contributors: rantum, danning.sui, bakabhai993
+    config:
+      tags: ['optimism','0x','dex_aggregator','dex','aggregator']
+    description: >
+        0x API erc20 swaps raw fills (including multihops) and also native swaps thru 0x exchange contracts (without using 0x API) on Ethereum
+    tests:
```

Comment:
missing seed test?

Reply:
there's no valid txns on optimism to include in a seed file yet